css/default.css

```diff
@@ -300,7 +300,7 @@ code pre {
   height: 22px;
   display: inline-flex;
   margin-left: 12px;
-  background-image: url('data:image/svg+xml,%3Csvg width='21' height='22' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg opacity='.9' fill='%235d80ea'%3E%3Cpath d='M19.798 2.162a4 4 0 0 1 0 5.657l-2.829 2.828a3.991 3.991 0 0 1-3.052 1.165.5.5 0 1 1 .055-.998 2.992 2.992 0 0 0 2.29-.874l2.829-2.828a3 3 0 0 0-4.243-4.243L12.02 5.697a2.992 2.992 0 0 0-.874 2.29.5.5 0 0 1-.999.056 3.991 3.991 0 0 1 1.166-3.053l2.828-2.828a4 4 0 0 1 5.657 0ZM2.121 19.838a4 4 0 0 1 0-5.657l2.829-2.828a3.991 3.991 0 0 1 3.053-1.165.5.5 0 1 1-.056.998 2.992 2.992 0 0 0-2.29.874l-2.829 2.828a3 3 0 0 0 4.243 4.243l2.828-2.828c.63-.629.92-1.466.874-2.29a.5.5 0 1 1 .999-.056 3.992 3.992 0 0 1-1.165 3.053l-2.829 2.828a4 4 0 0 1-5.657 0Z'/%3E%3Cpath d='M6.717 14.535a.5.5 0 1 0 .708.707l7.778-7.778a.5.5 0 1 0-.707-.707l-7.779 7.778Z'/%3E%3C/g%3E%3C/svg%3E');
+  background-image: url("data:image/svg+xml,%3Csvg width='21' height='22' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cg opacity='.9' fill='%235d80ea'%3E%3Cpath d='M19.798 2.162a4 4 0 0 1 0 5.657l-2.829 2.828a3.991 3.991 0 0 1-3.052 1.165.5.5 0 1 1 .055-.998 2.992 2.992 0 0 0 2.29-.874l2.829-2.828a3 3 0 0 0-4.243-4.243L12.02 5.697a2.992 2.992 0 0 0-.874 2.29.5.5 0 0 1-.999.056 3.991 3.991 0 0 1 1.166-3.053l2.828-2.828a4 4 0 0 1 5.657 0ZM2.121 19.838a4 4 0 0 1 0-5.657l2.829-2.828a3.991 3.991 0 0 1 3.053-1.165.5.5 0 1 1-.056.998 2.992 2.992 0 0 0-2.29.874l-2.829 2.828a3 3 0 0 0 4.243 4.243l2.828-2.828c.63-.629.92-1.466.874-2.29a.5.5 0 1 1 .999-.056 3.992 3.992 0 0 1-1.165 3.053l-2.829 2.828a4 4 0 0 1-5.657 0Z'/%3E%3Cpath d='M6.717 14.535a.5.5 0 1 0 .708.707l7.778-7.778a.5.5 0 1 0-.707-.707l-7.779 7.778Z'/%3E%3C/g%3E%3C/svg%3E");
```

Comment:
Was this an intentional or a tooling change? Let’s agree on a standard shortly—in our CSS folder, I just refactored everything to use single quotes.